PHPCI/Plugin/CopyBuild.php

```diff
@@ -35,12 +36,22 @@ class CopyBuild implements \PHPCI\Plugin
      */
     public function __construct(Builder $phpci, Build $build, array $options = array())
```

Comment:
This is just reading in the parameters and setting defaults if they haven't been set, hence the high NPath complexity. In reality I can't see this being an issue.